saleor/order/views.py

```diff
@@ -100,6 +100,10 @@ def start_payment(request, order, variant):
             defaults=defaults)
         try:
             form = payment.get_form(data=request.POST or None)
+            order.create_history_entry(
```

Comment:
I know we used to create an empty entry when starting a new payment, but I'm really guessing if this entry does help with anything.

Reply:
I have the same thoughts with this one. I don't really know. Maybe in some cases but feels like hidden debug message?